### ISSUE TITLE
Foreign key CASCADE strategy

### ISSUE THREAD

**Issue by catan85** [adrianandrei-ca/staruml-postgresql#3]
It could be usefull if the user can set some tag in StarUML that on code generation are translated into the DELETE and UPDATE strategy for the foreign key. 
I.E. 
...  CONSTRAINT fk_events__machine_id FOREIGN KEY (machine_id)
      REFERENCES public.machines (machine_id) 
      ON UPDATE CASCADE ON DELETE CASCADE ....

**Comment by adrianandrei-ca** [comment]
Yeah, I am procrastinating on this one ... I would have preferred the relationships were used as reference instead of referenceTo field in properties editor; would have made an elegant solution but people would have to add them into the model. Adding the reference policy through tags is doable but I would like to think a little bit about it.
